.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,11 @@
 blank_issues_enabled: true
 contact_links:
-  - name: moja global Slack chat.
+  - name: moja global's Slack.
     url: https://join.slack.com/t/mojaglobal/shared_invite/zt-o6ta1ug0-rVLjAo460~d7JbZ~HpFFtw
-    about: Chat to us about moja global and ask general questions.
\ No newline at end of file
+    about: Talk to us about moja global and ask general questions.
```

Comment:
I don't see how this change helps

Reply:
I was following a template shared by @chicken-biryani  for updating config.yml
But I get that there's no change in semantic meaning of either one. Should I keep this or change it back to the initial term? @HarshCasper

Reply:
Hi, @utkershrajvenshi I believe it can be removed.
Once this PR is done I believe you can do the same across the projects.